docs/concepts-messaging.md

```diff
@@ -904,7 +904,7 @@ If there is a key attached to message, the messages will be routed to correspond
 
 ### Hashing scheme
 
-{@inject: javadoc:HashingScheme:/client/org/apache/pulsar/client/api/HashingScheme} is an enum that represents sets of standard hashing functions available when choosing the partition to use for a particular message.
+[HashingScheme](https://pulsar.apache.org/api/client/2.11.x/org/apache/pulsar/client/api/HashingScheme) is an enum that represents sets of standard hashing functions available when choosing the partition to use for a particular message.
```

Comment:
```suggestion
[HashingScheme](/api/client/org/apache/pulsar/client/api/HashingScheme) is an enum that represents sets of standard hashing functions available when choosing the partition to use for a particular message.
```

I think the pattern is as this. I may try to migrate the post replace logic to a preview time avialable TSX component.